src/gui/plugins/plotting/GazeboPlotting.hh

```diff
@@ -0,0 +1,81 @@
+#ifndef IGNITION_GUI_PLUGINS_PLOTTING_HH_
+#define IGNITION_GUI_PLUGINS_PLOTTING_HH_
+
+#include <ignition/gui/Application.hh>
+#include <ignition/gui/qt.h>
+#include <ignition/gazebo/EntityComponentManager.hh>
+#include "ignition/gazebo/components/Name.hh"
+#include "ignition/gazebo/components/Pose.hh"
+#include <ignition/gazebo/gui/GuiSystem.hh>
+#include <string>
+#include <iostream>
+#include <QTimer>
+#include "Transport.hh"
+#include <QStandardItemModel>
+
+#define TIMEOUT 100
+using namespace std;
+
+/// \brief Interface to Communicate with Qml, and to be a common code between the GuiPlugin and GazeboPlugin
+class PlottingInterface : public QObject
```

Comment:
Please put all the classes inside the `ignition::gazebo::version` namespace, see an example:

https://github.com/ignitionrobotics/ign-gazebo/blob/98d41cd7bd6a0f7c9293c0e1cbca81e59c5db0dc/src/gui/plugins/scene3d/Scene3D.hh#L45-L50

I'd also suggest removing `Gazebo` from `GazeboPlotting` since it will be part of the namespace.